ydb/tests/functional/canonical/canondata/test_sql.TestCanonicalFolder1.test_case_write_multi_write.sql-plan_/write_multi_write.sql.plan

```diff
@@ -18,17 +17,26 @@
                 },
                 {
                     "columns": [
-                        "Comment",
                         "Group",
                         "Name"
                     ],
-                    "type": "MultiUpsert"
+                    "type": "MultiErase"
```

Comment:
Wrong order???

Reply:
It's ok.
Original order also is different from sql
https://github.com/ydb-platform/ydb/blob/48ddc8cc3eb31718c820362a1af0031c5df07c5a/ydb/tests/functional/canonical/canondata/test_sql.TestCanonicalFolder1.test_case_write_multi_write.sql-plan_/write_multi_write.sql.plan#L8